packages/marko/src/runtime/components/registry-browser.js

```diff
@@ -1,51 +1,85 @@
+var complain = "MARKO_DEBUG" && require("complain");
 var defineComponent = require("./defineComponent");
+var loader = require("../../loader");
+var queueMicrotask = require("../queueMicrotask");
+var { tryHydrateComponent } = require("../components/init-components-browser");
```

Comment:
Since we do not use babel for the client side runtime, we cannot use object destructuring here.